cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -44,11 +63,98 @@ def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs
         except Exception: # nosec
             pass
         if frame_info is not None:
-            image_info[frame] = (frame_info['height'], frame_info['width'])
+            image_info[frame] = (frame_info["height"], frame_info["width"])
 
-    detect_dataset(temp_dir, format_name='yolo', importer=dm_env.importers.get('yolo'))
-    dataset = Dataset.import_from(temp_dir, 'yolo',
-        env=dm_env, image_info=image_info)
+    detect_dataset(temp_dir, format_name=format_name, importer=dm_env.importers.get(format_name))
+    dataset = Dataset.import_from(temp_dir, format_name,
+        env=dm_env, image_info=image_info, **(import_kwargs or {}))
     if load_data_callback is not None:
         load_data_callback(dataset, instance_data)
     import_dm_annotations(dataset, instance_data)
+
+
+@importer(name='YOLO', ext='ZIP', version='1.1')
+def _import_yolo(src_file, temp_dir, instance_data, **kwargs):
+    _import_common(
+        src_file,
+        temp_dir,
+        instance_data,
+        format_name="yolo",
+        **kwargs
+    )
+
+
+@exporter(name='YOLOv8 Detection', ext='ZIP', version='1.0')
+def _export_yolov8_detection(dst_file, temp_dir, instance_data, save_images=False):
+    _export_common(dst_file, temp_dir, instance_data, 'yolov8', save_images=save_images)
```

Comment:
I would recommend to use here 'yolov8_detection'. In the future, we can use yolov8 for exporting all annotations. All other variants have format suffix. I would add it for YOLOv8 Detection as well for consistency.

Reply:
renamed